src/wikigraph.cc

```diff
@@ -1,21 +1,26 @@
 #include "wikigraph.hpp"
-#include "utilities.hpp"
-#include <unordered_map>
+
 #include <fstream>
+#include <queue>
+#include <unordered_map>
+
+#include "utilities.hpp"
 
 // TODO: Construct from File
 WikiGraph::WikiGraph(const std::string& file_name) {
   // Reads in file stream.
   // Populates map.
 
   // open file
-  // get a line; decode the first item, decode the second item, add the second to adjlist of firsts
+  // get a line; decode the first item, decode the second item, add the second
+  // to adjlist of firsts
   std::string file = file_to_string(file_name);
   std::vector<std::string> lines;
   SplitString(file, '\n', lines);
-  lines.pop_back(); // our dataset has a line of empty space that we can discard
+  lines
+      .pop_back();  // our dataset has a line of empty space that we can discard
 
-  std::unordered_map<std::string, std::string> decoded; // memoize decoding
+  std::unordered_map<std::string, std::string> decoded;  // memoize decoding
```

Comment:
does this look better with two spaces? i feel like 1 is fine

Reply:
sure.